rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewTest.java

```diff
@@ -260,6 +264,32 @@
     o.assertNoMoreEvents();
   }
 
+  @TargetApi(M)
+  @SdkSuppress(minSdkVersion = M)
+  @Test @UiThreadTest public void scrollChangeEvents() {
```

Comment:
Yeah, sorry about that. I haven't run it on local M-emulator yet. Just start downloading the emulator image on a hacked wifi, might take a while...